pkgs/development/tools/cppclean/default.nix

```diff
@@ -0,0 +1,31 @@
+{ stdenv, lib, fetchFromGitHub, python3Packages }:
+
+with python3Packages;
+
+buildPythonApplication rec {
+  name = "cppclean-${version}";
```

Comment:
Missed it the first time, but since we are fetching an unstable git commit, [guidelines](https://nixos.org/nixpkgs/manual/#sec-package-naming) would recommend:
```nix
  name = "cppclean-unstable-${version}";
  version = "2018-05-12";
```